conda-forge.yml

```
@@ -1,5 +1,8 @@
 azure:
-  free_disk_space: true
+  free_disk_space:
+    - apt
+    - cache
+    - docker
   settings_linux:
     swapfile_size: 10GiB
```

We seem to be getting close to memory usage limits on CI in past builds. If that keeps happening, we may want to consider bumping the swap file size

```suggestion
    swapfile_size: 20GiB
```

---

Looks like the cleanup of Docker images helped. Not seeing this memory usage warning on new builds

---

Saw this in a couple recent builds again. Sadly the Azure logs are not still linkable

In any event, going to try this to see if it can help